hack/.golint_failures

```diff
@@ -144,6 +144,7 @@ pkg/controller/certificates
 pkg/controller/certificates/approver
 pkg/controller/certificates/signer
 pkg/controller/cloud
+pkg/controller/clusterroleaggregation
```

Comment:
I added this because I had one failure about `pkg/controller/clusterroleaggregation/clusterroleaggregation_controller.go:42:6: type name will be used as clusterroleaggregation.ClusterRoleAggregationController by other packages, and that stutters; consider calling this Controller`, but I think the specific name is substantially better given that **everything** is a controller in the files this will be called from.